### PR TITLE
Accessibility changes

### PR DESCRIPTION
**Pull request by Amtresu**
Made text responsive to accommodate browser text size. Changed colors to make seeing them easier.
1. Page has a title.
2.  All images have alt
3. Page has one h1, and subsequent appropriate headings.
4. Color scheme provided by FIGMA, did update gray text font as it was hard to see against white background. 
5. Updated font to make it more responsive.
6. All clickables can be tabbed to.
7. There is no moving content.
8. There is no multimedia
9. Changed header order so nothing would be out of order.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -353,9 +353,9 @@ <h2 id="greeting_2">About<br>Myself</h2>
             <p class="about_me">I’m a software developer! I can help you build a<br> product , feature or website Look through some of my<br>work and experience! If you like what you see and have<br> a project you need coded, don’t hestiate to contact me.</p>
         
 
-        <h4>
+        <h3>
```
Comment (Amtresu) [inline index.html:356]: This was done so that headers would be in descending order.

File: style.css [hunk 2]
```diff
@@ -144,7 +143,7 @@
     display: flex;
     gap: 15px;
     margin-top: 5px;
-    color: #7a869a;
+    color: #21201E;
```
Comment (Amtresu) [inline style.css]: Changed to a darker gray so it would be more visible

File: style.css [hunk 3]
```diff
@@ -315,16 +314,16 @@
 
   .contact_header {
     font-weight: 700;
-    font-size: 40px;
+    font-size: 300%;
```
Comment (Amtresu) [inline style.css:315]: Made text responsive.

**Review by omarsalem7 — APPROVED** [review]
Hi @Amtresu 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**